lunavl/sdk/luna_faces.py

```diff
@@ -392,13 +392,27 @@ def __init__(
         Args:
             detectorType: detector type
             faceEngine: face engine for detector and estimators
+            estimationSettings: settings for detection
         """
-        if faceEngine is not None:
+        if faceEngine is None:
+            if not hasattr(self, "faceEngine"):
+                raise RuntimeError(f"Initialize the '{self.__class__.__name__}' first or pass faceEngine")
+        else:
             self.faceEngine = faceEngine
-            self.estimatorsCollection = FaceEstimatorsCollection(faceEngine=self.faceEngine)
```

Comment:
estimatorsCollection  has other faceEngine now. I think that need return  this line